atomicfu/src/commonMain/kotlin/kotlinx/atomicfu/Trace.common.kt

```diff
@@ -60,12 +60,20 @@ public expect fun TraceBase.named(name: String): TraceBase
 public expect val traceFormatDefault: TraceFormat
 
 public open class TraceBase internal constructor() {
```

Comment:
This class is public and user-facing, but neither it nor its method has a documentation